uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader/Steps/1000.php

```diff
@@ -113,19 +113,15 @@ private function up1000_setAvailableCountries() {
    * Sets Available Provinces to 'all provinces'
    */
   private function up1000_setAvailableProvinces() {
-    // ToDo : Fetch Provinces via API after upgrading to newer civicrm version since it is not available on civicrm 4.7.9
-    $tableName = CRM_Core_DAO_StateProvince::getTableName();
-    $query = CRM_Core_DAO::executeQuery("SELECT id FROM {$tableName}");
+    $countries = civicrm_api3('Country', 'get', ['sequential' => 1]);
```

Comment:
Are you sure we should use the Country API instead of StateProvince? The previous version of the code was fetching the data using the StateProvince BAO